index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Sameed - Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Poppins:wght@500&display=swap" rel="stylesheet">
+</head>
+
+
+
+<body background="./Images for portfolio/header-shapes mobile@2x.svg">
```

Comment:
- [ ] Good job adding the background as suggested by the previous reviewer 👍🏽 .Kindly ensure your background looks exactly like the one on the Figma design. In real life project, you will be expected to replicate designs to be pixel-perfect, so it will be nice to put that into practice. 👍🏽 

Hint: Kindly add your background image in your CSS file so as to make your codebase easy to read and clear. You can also adjust the positioning of your existing background image.

Your design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212498398-0c6423e2-9574-41e7-ab21-4fbeed1c8f63.png)
 
FIgma design 👇🏽 
![image](https://user-images.githubusercontent.com/63473584/212498469-51b00046-01d9-4933-8041-f3695bca3467.png)

Reply:
Can you please guide me a bit more? Like do I have to add 2 background images?

Reply:
@not really, just uae `background-position` css property  to position the background in your css file